### ISSUE TITLE
Import/Export Tables/Databases in .gz format

### ISSUE THREAD

**Issue by Dwza** [HeidiSQL/HeidiSQL#1954]
**Is your feature request related to a problem? Please describe.**
Mostly large sql dumps are passed in compressed `sql.gz` or `sql.tar.gz` format. These files have to be extracted befor importing them.

**Describe the solution you'd like**
For export it would be wonderful if we could select `gz` as a compression format (_equal to zip_).
For Import, HSQL should recognize the filetype, extract it and than go on with the regular process (_what ever was selected_).

**Describe alternatives you've considered**
This is a **nice to have** feature request that was obviously asked before. At least when searching the internet :) How ever, you could do everything manually with e.g. 7zip. But just for the convenience, I (_and may a lot other people_) would love to have this "easy life" improvement.

**Comment by barsk** [comment]
+1 on this feature request. The current support for ZIP export is nice but to complete the cycle a way to import zip archives would be a great improvement. Gzip is not a container as ZIP is so it should be much easier to implement I think. At least it is really easy in Java, hopefully in Delphi as well.

**Comment by ansgarbecker** [comment]
No, ZIP is easier to implement in Delphi, as there is a native wrapper for that, even with large file support. Perhaps because ZIP is more spread in the Windows world, and Delphi's tradition is also Windows. Saving GZ files requires some 3rd party library.

**Comment by barsk** [comment]
I can live fine with ZIP import! :)

**Comment by Dwza** [comment]
I mean this could be a dependance... the option is deactivated, if you try to activate it, it should try to search for 7-zip. If not installed msgbox or such thing.

**Comment by ansgarbecker** [comment]
Will convert this to "import zip files", which better meats the given possibilities of Delphi and Lazarus, than the .gz format.
